vividus/src/main/java/org/vividus/bdd/expression/RoundExpressionProcessor.java

```diff
@@ -25,26 +25,29 @@
 
 import javax.inject.Named;
 
-import org.apache.commons.lang3.EnumUtils;
 import org.apache.commons.lang3.math.NumberUtils;
+import org.jbehave.core.steps.ParameterConverters.FluentEnumConverter;
 
 @Named
 public class RoundExpressionProcessor extends AbstractExpressionProcessor<String>
 {
     private static final Pattern ROUND_EXPRESSION_PATTERN;
 
+    private static FluentEnumConverter fluentEnumConverter;
+
     static
     {
-        String roundingModes = Stream.of(RoundingMode.values()).map(e -> e.name().toLowerCase())
-                .collect(Collectors.joining("|"));
+        String roundingModes = Stream.of(RoundingMode.values()).map(e -> e.name().toLowerCase()
+                .replaceAll("_", "[ |_]")).collect(Collectors.joining("|"));
         String pattern = String.format("^round(?:\\((-?\\d+(?:\\.\\d*)?(?:[Ee]+(?:-|\\+)?\\d+)?)(?:,\\s*(\\d+))?"
                 + "(?:,\\s*(%s))?\\))$", roundingModes);
         ROUND_EXPRESSION_PATTERN = Pattern.compile(pattern, Pattern.CASE_INSENSITIVE);
     }
 
-    public RoundExpressionProcessor()
+    public RoundExpressionProcessor(FluentEnumConverter fluentEnumConverter)
     {
         super(ROUND_EXPRESSION_PATTERN);
+        RoundExpressionProcessor.fluentEnumConverter = fluentEnumConverter;
```

Comment:
it's not a good way to initialize static field in the constructor, you can make the field non-static and pass it to `RoundExpression` constructor